### PR TITLE
Docs: Edit the javascript bitwise operators entry

### PR DESCRIPTION
**Pull request by braanj**
### Description

Rewrite the description and shorten it to less than 160 character. Modify and add operators definitions and add new examples.

### Issue Solved

#5385 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by braanj** [comment]
Hey @SaviDahegaonkar thanks for the review. The PR has been updated with your suggestions.

File: content/javascript/concepts/bitwise-operators/bitwise-operators.md [hunk 1]
```diff
@@ -25,65 +25,86 @@ JavaScript uses the following bitwise operators:
...
 | `>>>`    | Zero fill right shift | Pushes zeros in from left, rightmost bits fall off.                                   |
 
-## Examples
+## AND
 
-### AND
+The `&` operator performs a bitwise AND on two numbers. The result is 1 only where both bits are 1.
 
-```javascript
+### Examples
```
Comment (SaviDahegaonkar) [inline content/javascript/concepts/bitwise-operators/bitwise-operators.md:32]:
```suggestion
### Example
```

File: content/javascript/concepts/bitwise-operators/bitwise-operators.md [hunk 2]
```diff
@@ -25,65 +25,86 @@ JavaScript uses the following bitwise operators:
...
 
-### NOT
+## NOT
+
+Because integers are stored in two's complement (to change the sign, invert the binary digits and add one), a `~` operation will change the sign of the number and change the absolute value by one.
 
-Because integers are stored in two's complement
-(to change the sign, invert the binary digits and add one)
-a `~` operation will change the sign of the number and change the absolute value by one.
+### Examples
```
Comment (SaviDahegaonkar) [inline content/javascript/concepts/bitwise-operators/bitwise-operators.md:74]:
```suggestion
### Example
```

File: content/javascript/concepts/bitwise-operators/bitwise-operators.md [hunk 3]
```diff
@@ -25,65 +25,86 @@ JavaScript uses the following bitwise operators:
...
+// 00000000000000000000000000010100 = 20
 ```
 
-### Signed right shift
+## Signed right shift
+
+The `>>` operator shifts the bits of a number to the right, pushing zeros in from the left. The rightmost bits fall off and are discarded.
 
-The `>>` operator preserves the sign of the operand by pushing copies of the leftmost bit in from the left.
+### Examples
```
Comment (SaviDahegaonkar) [inline content/javascript/concepts/bitwise-operators/bitwise-operators.md:105]:
```suggestion
### Example
```

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @braanj ,
LGTM! Thanks for Contributing to Docs :)


Thanks,
Savi

**Comment by braanj** [comment]
Hey @Sriparno08 please, could you review this PR.

**Comment by Sriparno08** [comment]
Hey @braanj, I'll review this PR soon.

**Review by Sriparno08 — APPROVED** [review]
Looks good, @braanj!

**Pull request merged**